server/start.go

```diff
@@ -167,9 +169,23 @@ is performed. Note, when enabled, gRPC will also be automatically enabled.
 				serverCtx.Logger.Info("starting ABCI without CometBFT")
 			}
 
-			return wrapCPUProfile(serverCtx, func() error {
+			err = wrapCPUProfile(serverCtx, func() error {
 				return start(serverCtx, clientCtx, appCreator, withCMT, opts)
 			})
+
+			serverCtx.Logger.Debug("received quit signal")
+
+			// wait for app.Close() and tmNode.Stop() to flush async db writes to disk
+			graceDuration, _ := cmd.Flags().GetDuration(FlagShutdownGrace)
+			if graceDuration > 0 {
+				serverCtx.Logger.Info("wait for database to flush", FlagShutdownGrace, graceDuration)
+				<-time.After(graceDuration)
+				serverCtx.Logger.Info("finished waiting for database to flush")
+			} else {
+				serverCtx.Logger.Info("do not wait for database to flush")
```

Comment:
imho, we should just remove this message.

Reply:
ok